coala_quickstart/green_mode/bear_settings.yaml

```diff
@@ -0,0 +1,94 @@
+type4:
```

Comment:
Add comments referring to what type1, type2, type3 and all imply

Reply:
And if I'm correct, this would be generated on the fly once the type checking approach is resolved, right?

Reply:
the commit message is somewhat confusing too.

Reply:
*"tried"* to fix it